### PR TITLE
Add the Staff Screen

### PR DESCRIPTION
**Pull request by rkowase**
## Issue
- none

## Overview (Required)
- Add the Staff Screen

## Links
- Similar Issue: https://github.com/DroidKaigi/conference-app-2018/issues/14
- Similar PR: https://github.com/DroidKaigi/conference-app-2018/pull/206

## Screenshot
Before | After
:--: | :--:
 | 
none |

### CODE REVIEW

**Comment by takahirom** [comment]
🆒

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -98,6 +99,14 @@
         <item quantity="other">%d Contributors</item>
     </plurals>
 
+    <!-- Staff -->
+    <string name="staff">Staff</string>
+    <string name="staff_avatar">staff avatar</string>
+    <plurals name="staff_header">
+        <item quantity="one">%d Staff</item>
+        <item quantity="other">%d Staffs</item>
```
[the remark this answers is not in the document]
Comment (rkowase) [reply app/src/main/res/values/strings.xml]: Thank you. I'll fix 🇺🇸

**Comment by takahirom** [comment]
👀

**Comment by takahirom** [comment]
How about creating local package and creating LocalJsonParser and Mapper in that package?

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/StaffViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                .toResult(schedulerProvider)
+                .toLiveData()
+    }
+
+    @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
+    fun onCreate() {
+        refreshStaff()
+    }
+
+    private fun refreshStaff() {
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/StaffViewModel.kt]: Is this method need? 🤔

**Comment by konifar** [comment]
Thanks
I thought this staff page should be merged to contributors page (maybe using TabLayout?).
But currently we don't have any design of it. So for now LGTM 👍

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/StaffViewModel.kt (hunk 2 above)
Comment (rkowase) [reply app/src/main/java/io/github/droidkaigi/confsched2018/presentation/staff/StaffViewModel.kt]: This method was not necessary. I fixed in https://github.com/DroidKaigi/conference-app-2018/pull/403/commits/a3064f5d4838d0bd2e3ce76b05a3fc46099dbac5

**Comment by rkowase** [comment]
> How about creating a local package and creating LocalJsonParser and Mapper in that package?

I created a `local` package in https://github.com/DroidKaigi/conference-app-2018/pull/403/commits/c94d98fbe8f96396df869dd9958f54584159a7d5
It might be better to create `StringExt.kt`. what do you think?

**Comment by konifar** [comment]
I think it's too much to create `StringExt.kt` to parse json.
So current implementation is fine for me.

**Comment by konifar** [comment]
Let me merge 😄

**Pull request merged**